src/renderer/components/FlippersView.tsx

```diff
@@ -154,7 +154,7 @@ const StyledPostion = styled.p`
 `
 
 // const StyledMotorBusVoltage = styled.p`
-  // grid-area: t;
+// grid-area: t;
 // `
```

Comment:
Are these commented out lines meant to still be there?

Reply:
Ok, I understand the idea. In that case, what you just wrote should be recorded somewhere instead. As is, these lines of code don't represent the intent and are deep inside the code where we won't see them.
Either you commit these lines uncommented on a new branch for that project or the task on Teams should be updated / a new one created. Either way, this PR should be merged without those lines in, then we'll do the figuring out.